DEVELOPER_GUIDE.md

```diff
@@ -14,6 +14,7 @@
     - [IntelliJ IDEA](#intellij-idea)
     - [Visual Studio Code](#visual-studio-code)
     - [Eclipse](#eclipse)
+    - [JetBrains Gateway](#jetbrains-gateway)
```

Comment:
Shouldn’t this be just under https://github.com/opensearch-project/OpenSearch/pull/1944/files#diff-d5555724c2ebca1b1f9be92a74c90e84c2dcbc76631148801024e24372b7edc6R14 and no separate index for the same?

Reply:
Yes, it makes sense to add it under remote development for IntelliJ